### PR TITLE
Docs: Fix broken links and typos, propose clarity improvements

### PR DESCRIPTION
**Pull request by IAmHopp**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~~Added/Updated related documentation.~~
- ~~Added/Updated related tests.~~

## Short description of the change(s)

Since there were no objections in #1216, I'm going through each hint doc and checking for typos and suggesting minor changes for improved readability. If there are any objections, please let me know and I'll abandon the PR.

Files to review:

- [x] hint-amp-validator
- [x] hint-apple-touch-icons
- [x] hint-axe
- [x] hint-babel-config
- [x] hint-babel-config/is-valid
- [x] hint-content-type
- [x] hint-disown-opener
- [x] hint-highest-available-document-mode
- [x] hint-html-checker
- [x] hint-http-cache
- [x] hint-http-compression
- [x] hint-https-only
- [x] hint-image-optimization-cloudinary
- [x] hint-manifest-app-name
- [x] hint-manifest-exists
- [x] hint-manifest-file-extension
- [x] hint-manifest-is-valid
- [x] hint-meta-charset-utf-8
- [x] hint-meta-theme-color
- [x] hint-meta-viewport
- [x] hint-minified-js
- [x] hint-no-bom
- [x] hint-no-broken-links
- [x] hint-no-disallowed-headers
- [x] hint-no-friendly-error-pages
- [x] hint-no-html-only-headers
- [x] hint-no-http-redirects -- See #1216.
- [x] hint-no-protocol-relative-urls
- [x] hint-no-p3p
- [x] hint-no-vulnerable-javascript-libraries
- [x] hint-performance-budget
- [x] hint-sri -- See #1210.
- [x] hint-ssllabs
- [x] hint-strict-transport-security
- [x] hint-stylesheet-limits
- [x] hint-typescript-config
- [x] hint-typescript-config/consistent-casing
- [x] hint-typescript-config/import-helpers
- [x] hint-typescript-config/is-valid
- [x] hint-typescript-config/no-comments
- [x] hint-typescript-config/strict
- [x] hint-typescript-config/target
- [x] hint-validate-set-cookie-header -- See #1209.
- [x] hint-webpack-config
- [x] hint-webpack-config/config-exists
- [x] hint-webpack-config/is-installed
- [x] hint-webpack-config/is-valid
- [x] hint-webpack-config/module-esnext-javascript
- [x] hint-webpack-config/modules-false-babel
- [x] hint-webpack-config/no-devtool-in-prod
- [x] hint-x-content-type-options

Once done:
- [x] Additional passthrough [specific for broken links]
- [x] Final passthrough
- ~~Check files changed~~ -- Vast majority of commits were merged ahead of time yesterday, so this wouldn't be valuable anymore.

### CODE REVIEW

**Comment by alrra** [comment]
@IAmHopp Thank you so much for doing this! 🎉 Let us know if we can help with anything!

**Comment by IAmHopp** [comment]
Sure thing! Appreciate the support.

I'll continue tomorrow. With any luck I won't swallow a bunch of commits accidently as I did today.

**Comment by IAmHopp** [comment]
@alrra I'm about halfway through a final passthrough, but so far I noticed two things I need a bit more guidance on:

- In hint-axe, the quote from [WAI’s Introduction to Web Accessibility](https://www.w3.org/WAI/intro/accessibility.php) seems to be out-of-date. I went looking because of a section in the last paragraph, "[...] that web i accessibility [...]", that seemed a bit odd. What do you guys think? Should this quote be updated? If not, any idea what's up with that out-of-place "i'?
- In hint-no-p3p, the sentence "Others it was typos and errors in the tokens (but still invalid policies)" at the end of the fourth paragraph seems odd to me as well. Is that meant to say "Others had typos and errors in the tokens"? Also, I'm not sure that information in brackets is necessary. Aren't configurations with typos and token errors also invalid?

I'm not expecting to run into many other issues like these two, so I should be done with the rest tomorrow.

**Comment by alrra** [comment]
@IAmHopp Great work!! 🎉 Thank you again for doing this!!

>  If not, any idea what's up with that out-of-place "i'?

Just a typo, fixed in https://github.com/webhintio/hint/commit/1307d34ac6813699f915e0e74c416544ae146daa#diff-8ac9c116fee2ce72502a2354537fd108R27.

---

> In hint-no-p3p, the sentence "Others it was typos and errors in the tokens (but still invalid policies)" at the end of the fourth paragraph seems odd to me as well. Is that meant to say "Others had typos and errors in the tokens"? Also, I'm not sure that information in brackets is necessary. Aren't configurations with typos and token errors also invalid?

Yes, something such as _"Others had typos and errors in the tokens"_ sounds ok to me. Also, yes, the information in the brackets doesn't seem necessary.

---

> I'm not expecting to run into many other issues like these two, so I should be done with the rest tomorrow.

I went to all your commits, made a few changes, and merged them into `master`. Feel free to continue pushing to this PR and I'll pull the new commits in (or if you are comfortable, rebase, if not, don't stress to much about it, I'll handle the merging part).

(Note to self: last commit merged 86de0f730aec34b7cf3ab20a1553cfc1f0698608).

---

In the future, can you use `Docs: <Title>` for the title of the [commit messages](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)? Thanks!

**Comment by molant** [comment]
> If not, any idea what's up with that out-of-place "i'?

Also the new section in the link is [Accessibility is Important for Individuals, Businesses, Society](https://www.w3.org/WAI/fundamentals/accessibility-intro/#important) in case we want a more up to date quote. 

Thanks a lot for your work @IAmHopp !

**Comment by alrra** [comment]
@IAmHopp Your changes are now live! 🎉 (e.g.: [change](https://github.com/webhintio/hint/commit/7aa393f2e8a6b2e1675c0efcb68320b2dbb83eb0) ➡️ [site](https://webhint.io/docs/user-guide/hints/hint-manifest-is-valid/)).

**Comment by IAmHopp** [comment]
Amazing! Glad I could help. I'll rebase and wrap this up tomorrow.

**Comment by IAmHopp** [comment]
Alright, guys, One last case I want to report: The "Can the hint be configured?" section in hint-meta-viewport looks sort of broken. The information that's there seems to belong elsewhere, which, if true, would leave it devoid of content, of course. I think one of you should take a look.

As for the other files, I think I'm done. Let me know if you need my help elsewhere.

**Comment by molant** [comment]
Thanks a lot @IAmHopp!

> Let me know if you need my help elsewhere.

All the issues that don't have someone assigned are available that wants to do it and we can help in the process 😊
Do you want to do some coding or continue with documentation? It will be good to review the other sections of the user and contributor guides and make sure they make sense and everything is well structured.

Re the `meta-viewport` rule, maybe we can have something similar to:

> This hint can not be configured. It takes into consideration the targeted browsers, and if no versions of Safari for iOS < 9 are included, it will not require `initial-scale=1`.

**Comment by alrra** [comment]
> The information that's there seems to belong elsewhere

The information belongs there, but yes, it can be improved.

> As for the other files, I think I'm done.

I'll start reviewing and merge this PR. Again, thank you so much for doing this, we sincerely appreciate it!.

---

@IAmHopp Did you learn anything new going through the docs?

File: packages/hint-meta-charset-utf-8/README.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ As for the charset meta tag, always use `<meta charset="utf-8">` as:
   example]. The same may be true for other agents (non-browsers) that
   may scan/get the content and may not have the alias.
 
-* Must be inside the `<head>` element and [within the first 1024
+* It must be inside the `<head>` element and [within the first 1024
```
Comment (alrra) [inline packages/hint-meta-charset-utf-8/README.md]: `It needs to`

**Pull request closed without merging**